player/scripting.c

```diff
@@ -255,14 +255,24 @@ typedef int (*mpv_open_cplugin)(mpv_handle *handle);
 static int load_cplugin(struct mpv_handle *client, const char *fname)
 {
     int r = -1;
+    char *err;
```

Comment:
Completely pointless to put the decl here, instead of where it's define. (And even though that "duplicates" the decl, it'd be one line of code less in total.)